### ISSUE TITLE
Use C

### ISSUE THREAD

**Issue by sleeplessai** [open-mmlab/mmdetection#2614]
In data configuration, I find that COCO validation set plays the role of test set to evaluate the model mAP. So I think the true mAP on the COCO test set may submit to CodaLab to eval. It's that right? [https://competitions.codalab.org/competitions/5181#participate](https://competitions.codalab.org/competitions/5181#participate
)

> data = dict(
>     imgs_per_gpu=2,
>     workers_per_gpu=2,
>     train=dict(
>         type=dataset_type,
>         ann_file=data_root + 'annotations/instances_train2017.json',
>         img_prefix=data_root + 'train2017/',
>         pipeline=train_pipeline),
>     val=dict(
>         type=dataset_type,
>         ann_file=data_root + 'annotations/instances_val2017.json',
>         img_prefix=data_root + 'val2017/',
>         pipeline=test_pipeline),
>     test=dict(
>         type=dataset_type,
>         ann_file=data_root + 'annotations/instances_val2017.json',
>         img_prefix=data_root + 'val2017/',
>         pipeline=test_pipeline))
